src/test/java/hudson/plugins/plot/CSVSeriesTest.java

```diff
@@ -104,18 +118,227 @@ public void testCSVSeriesWithTrailingSemicolonDoesntCreateExtraneousPoint() {
         testSeries(series, file, "", "csv");
 
         // load the series.
-        List<PlotPoint> points = series.loadSeries(workspaceRootDir, 0, System.out);
+        List<PlotPoint> points = series.loadSeries(workspaceRootDir,
+                0,
+                System.out);
+        LOGGER.info("Got " + points.size() + " plot points");
+        testPlotPoints(points, 8);
+    }
+
+    @Test
+    public void testCSVExclusionValue_IntArray() {
+        CSVSeries series = new CSVSeries("test.csv",
+                null,
+                "EXCLUDE_BY_STRING",
+                "123,345",
+                false);
+        List<PlotPoint> points = series.loadSeries(workspaceRootDir,
+                0,
+                System.out);
         LOGGER.info("Got " + points.size() + " plot points");
         testPlotPoints(points, 8);
     }
 
+    @Test
+    public void testCSVInclusionValue_IntArray() {
+        CSVSeries series = new CSVSeries("test.csv",
+                null,
+                "INCLUDE_BY_STRING",
+                "123,345",
+                false);
+        List<PlotPoint> points = series.loadSeries(workspaceRootDir,
+                0,
+                System.out);
+        LOGGER.info("Got " + points.size() + " plot points");
+        testPlotPoints(points, 0);
+    }
+
+    @Test
+    public void testExcludeByRegexInAList() {
+        CSVSeries series = new CSVSeries("test_regex-webstatistics.csv",
+                null,
+                "EXCLUDE_BY_STRING",
+                "\"HTTP_[4,5]\\d{2}\",\"Hits\",\"Throughput\",\"RunId\",\"Trend Measurement Type\"",
+                false);
+        List<PlotPoint> points = series.loadSeries(workspaceRootDir,
+                0,
+                System.out);
+        LOGGER.info("Got " + points.size() + " plot points");
+        testPlotPoints(points, 3);
+    }
+
+    @Test
+    public void testIncludeBySingleRegexWithComma() {
+        CSVSeries series = new CSVSeries("test_regex-webstatistics.csv",
+                null,
+                "INCLUDE_BY_STRING",
+                "\"HTTP_[2,3]\\d{2}\"",
+                false);
+        List<PlotPoint> points = series.loadSeries(workspaceRootDir,
+                0,
+                System.out);
+        LOGGER.info("Got " + points.size() + " plot points");
+        testPlotPoints(points, 3);
+    }
+
+    @Test(expected = java.util.regex.PatternSyntaxException.class)
+    public void testIncludeBySingleRegexWithComma_unescaped_shouldFail() {
+        CSVSeries series = new CSVSeries("test_regex-webstatistics.csv",
+                null,
+                "INCLUDE_BY_STRING",
+                "HTTP_[2,3]\\d{2}",
+                false);
+        List<PlotPoint> points = series.loadSeries(workspaceRootDir,
+                0,
+                System.out);
+        LOGGER.info("Got " + points.size() + " plot points");
+        testPlotPoints(points, 0);
+    }
+
+    @Test
+    public void testIncludeTestuserByRegex() {
```

Comment:
Test name should not contain the particular column name (`Testuser`) but rather should be something like `testIncludeBySuffixRegex` (here and below).

Reply:
You are right. I will fix the name of the test case(s)